### DISCUSSION TITLE
QUIC 和 UDP 究竟应该怎么设置才比较合理

### DISCUSSION THREAD

**Discussion by chshouyu** [Openwrt-Passwall/openwrt-passwall2#577] [Q&A]
<img width="590" alt="image" src="https://github.com/xiaorouji/openwrt-passwall2/assets/1304342/eba9108d-30be-4362-bf64-11b557756c85">

目前这样设置的，协议使用的 vmess，看到很多插件默认都屏蔽了 QUIC，这么做真的有好处吗？
另外末尾的 UDP 大家都是怎么设置的？

**Comment by ghost** [comment]
如果您使用的代理組件爲 sing-box/naive proxy/tuic 中的任意一種，無論代理出口是否能UDP轉發，本地都是不能正常代理UDP流量的，多多少少會有bug。本地關掉或者不關掉實際上用起來一樣，UDP流量都不通，或者幾乎不正常。

但如果您使用的不是上述任一一種，且代理出口能UDP轉發，則最好不要關閉，這會影響您的正常使用，例如遊戲或者語音電話應用。

**Reply by Travel2Here** [reply]
最后结论看懵了，楼主图里的关闭不是指的不启用相关规则吗？相当于QUIC UDP都走默认线路。
怎么跟是否使用UDP有关了？所以最佳设定是什么呢？

**Comment by HSK077** [comment]
https://burgess-t.cn:35006/index.php/2024/07/30/617.html
